core/rspamd/conf/antivirus.conf

```diff
@@ -4,5 +4,8 @@ clamav {
   symbol = "CLAM_VIRUS";
   type = "clamav";
   servers = "{{ ANTIVIRUS_ADDRESS }}";
+  {% if ANTIVIRUS_ACTION|default('discard') == 'reject' %}
```

Comment:
Why not just display the action so people can  also specify discard or other valid actions?

Reply:
Because discard (and all other) is handled differently (in dovecot, later in the mail pipeline). discard is the only action directly taken by rspamd.

Reply:
My bad :x